### ISSUE TITLE
"'NoneType' object has no attribute 'attrs'" error when activating some books

### ISSUE THREAD

**Issue by SR-G** [igbt6/Packt-Publishing-Free-Learning#115]
I have from time to time this error : 

```
Today's free Packt ebook grabbing has failed with exception: 'NoneType' object has no attribute 'attrs'!
```

Logs : 
```
[INFO] - Creating session...
[INFO] - Session created, logged in successfully!
[INFO] - Start grabbing eBook...
[INFO] - Captcha detected. Trying to solve it using anti-captcha.com.
[INFO] - TaskId created: 67762126
[INFO] - Waiting for completion of the task: 67762126...
[SUCCESS] - Solution found for task: 67762126
[SUCCESS] - eBook: '****************' has been successfully grabbed!
[INFO] - Retrieving complete information for '****************
[ERROR] - Exception occurred 'NoneType' object has no attribute 'attrs'
[INFO] - Sending email from **************** to **************** ...
[INFO] - Email to **************** has been succesfully sent
```

In the packt website > account > My Ebooks >the corresponding book seems to be correctly grabbed this day.

**Comment by mjenczmyk** [comment]
Thanks for submitting this error! Personally I don't use `-l` parameter so I've never seen this (the script lacks testing though).

OK, I see - it succeeded in grabbing the ebook, but failed on fetching ebook's data. It couldn't find URL for downloading code files, because there are none, so there is no `<a>` inside button (which is disabled). Did it happen with "Mastering Windows PowerShell Scripting - Second Edition" today? Did other books with which the script failed in this way also didn't have code files?

**Comment by SR-G** [comment]
I'm only using it since ten days. It has failed two times : 
- 18/11/2018 "Mastering Windows PowerShell Scripting"
- 08/11/2018 "Understanding software"
Indeed it seems both of these books don't have any code attached to them.

**Comment by mjenczmyk** [comment]
OK, the problem is that script tries to find code files URL even if it doesn't exist. Do you want to make a pull request or should I?

**Comment by mjenczmyk** [comment]
https://github.com/igbt6/Packt-Publishing-Free-Learning/pull/117

Here is a fix for this issue.

**Comment by igbt6** [comment]
Fixed at #117
